ncm-xrootd/src/main/pan/components/xrootd/schema.pan

```diff
@@ -176,11 +176,18 @@ type ${project.artifactId}_component_fed_options = {
   "siteName" ? string
 };
 
+function is_${project.artifactId}_logKeep = {
+  if (is_long(ARGV[0])) return(true);
```

Comment:
deprecate `long`? `long` or string is probably worse then only regexp'ed string.